geode-core/src/main/java/org/apache/geode/internal/cache/BucketRegion.java

```diff
@@ -1261,7 +1333,14 @@ void basicUpdateEntryVersion(EntryEventImpl event) throws EntryNotFoundException
   }
 
   protected void distributeUpdateEntryVersionOperation(EntryEventImpl event) {
-    new UpdateEntryVersionOperation(event).distribute();
+    UpdateEntryVersionOperation op = new UpdateEntryVersionOperation(event);
+    long viewVersion = -1;
+    try {
+      viewVersion = op.startOperation();
```

Comment:
For cases like this one it would be nice to move the try/finally to a new method in DistributedCacheOperation and just invoke that method.  Maybe call it distribute()???

```
public void distributed() {
  long token;
  try {
    token = startOperation();
    _distribute(); // private method
  } finally {
    endOperation(token);
  }
}
```
If you do this you can revert the changes you've made to a lot of classes.